server/chat-plugins/chatlog.ts

```diff
@@ -366,7 +383,38 @@ export const pages: PageTable = {
 };
 
 export const commands: ChatCommands = {
-	chatlog(target, room, user) {
-		this.parse(`/join view-chatlog-${room.roomid}--today`);
+	chatlog: {
+		search(target, room, user) {
```

Comment:
This will fail if you have a room named `search`.

I'm not sure how to best work around this; maybe have a separate `/chatlogroom [roomid], [search]` command, and then `/chatlog [search]`?

Reply:
i could just rename the '' command to 'view'?

Reply:
renamed it, removed '' altogether.

Reply:
...I think that's worse, but whatever.

`/chatlog` by itself still needs to show the current room's log.

Reply:
Also how are you going to search deleted room logs?